src/App.js

```diff
@@ -1,26 +1,42 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from "react";
+import "./App.css";
```

Comment:
Styling is a large part of frontend.
I suggest seeing methodologies and tools for it.
For my projects, I usually use css-modules or styled-components.
CRA and Material UI support css-modules.